rune/libenclave/internal/runtime/pal/pal_linux.go

```diff
@@ -44,10 +45,10 @@ func (pal *enclaveRuntimePal) Init(args string, logLevel string) error {
 	no_epm := true
 	if !strings.Contains(args, "no-epm") {
 		no_epm = false
-	       /* enclaveinfo.Layout retrieves from /proc/pid/mmaps, in file
+		/* enclaveinfo.Layout retrieves from /proc/pid/mmaps, in file
 		* mmaps /dev/sgx/enclave mmaping address is sorted from low
 		* address to high one. So layout[0].Addr will be minimum.
-		*/
+		 */
```

Comment:
Plz move these modifications to another commit.

Reply:
> Plz move these modifications to another commit.

Done, thanks.